src/UI/components/map/layers/overlayList.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';

```

Comment:
This file is very similar to the `baseMapList` file - can they be merged into one?